<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScreen</class>
 <widget class="QWidget" name="LoginScreen">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>800</width><height>600</height></rect>
  </property>
  <property name="minimumSize">
   <size><width>800</width><height>600</height></size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_titulo">
     <property name="font">
      <font><pointsize>22</pointsize><bold>true</bold></font>
     </property>
     <property name="text">
      <string>Need for Speed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>

   <item row="1" column="0">
    <widget class="QLabel" name="label_nombre">
     <property name="text"><string>Nombre:</string></property>
     <property name="alignment"><set>Qt::AlignRight|Qt::AlignVCenter</set></property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edit_nombre"/>
   </item>

   <item row="2" column="0">
    <widget class="QLabel" name="label_ip_servidor">
     <property name="text"><string>IP Servidor:</string></property>
     <property name="alignment"><set>Qt::AlignRight|Qt::AlignVCenter</set></property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edit_ip_servidor">
     <property name="placeholderText"><string>localhost o 127.0.0.1</string></property>
    </widget>
   </item>

   <item row="3" column="0">
    <widget class="QLabel" name="label_puerto">
     <property name="text"><string>Puerto:</string></property>
     <property name="alignment"><set>Qt::AlignRight|Qt::AlignVCenter</set></property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edit_puerto">
     <property name="placeholderText"><string>p.ej. 5555</string></property>
    </widget>
   </item>

   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="btn_conectar">
     <property name="text"><string>Conectar</string></property>
    </widget>
   </item>

   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_error">
     <property name="styleSheet"><string notr="true">color: red;</string></property>
     <property name="text"><string/></property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
